### DISCUSSION TITLE
青春洋溢的校園路跑

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2685] [General]
校園路跑剛剛結束，好久沒看到這麼多人一起運動，那些跑在最前面的都好厲害，有些年紀看起來都蠻大的，真的是老當益壯

**Comment by CeBru13** [comment]
我當初跑完一圈就累得半死，完全沒辦法跟上第一批人，不過我有全程跑完我很開心

**Comment by forestdew507** [comment]
已經忘了大一自己怎麼完賽的，今年啃天氣還沒轉涼前練 3-5k 的老本，也有全程跑完，不過不得不說蠻累的XD
還是覺得有練有差，雖然我還是找不大到長距離跑這件事情本身哪裡有趣，但它絕對能達到運動的心肺標準，單就這點好像還蠻值得繼續跑的。不過戶外練還真的蠻看天氣，低溫風大的話懶惰都戰勝了一切哈哈。

**Comment by christaaah** [comment]
當時是直接起床簡單準備一下就跑了，現在想起來其實還蠻不好的，天氣也算蠻冷的，應該要好好的熱身並些吃一點點東西後再開始運動，不過跑完還是蠻舒服的，雖然也不想再來一次就是了。

**Comment by kzx-creat** [comment]
大一的時候有跑過一次，半跑半走的完成整個路跑，還記得當時怎麼跑都跑不到終點的感覺.....現在要我去跑會死，體力下降很多，青春不洋溢:/。

**Comment by Mminniehuang** [comment]
雖然我這次沒有參加路跑，但是在慢跑體育課又體驗了一次。
大一的時候明明感覺路程超級漫長，本來很生無可戀的，沒想到重新體驗一次的感受沒有想像中累，邊走邊跑邊聊天的情況下也很快就結束了。我想可能是人口密度沒有那麼高，而且那天天氣也很不錯的關係😆

**Comment by qiu0109** [comment]
大一的時候完全是邊跑邊走慢慢的完成比賽，我覺得有朋友在旁邊一起跑會感覺比較快樂，雖然邊聊天邊跑會真的很喘但還蠻有趣的。聽說這次路跑有學弟妹扮成蟑螂跟郵筒，如果學校可以另外頒個服裝創意獎應該會挺好玩的。

**Comment by yue083139** [comment]
之前大一路跑我基本上也是全程跑完，明明平常感覺就是跑不到這麼長的距離，或許是感覺到跟大家「一起」有某個目標的感覺吧，當天在跑其實感覺挺順的，就跟著人流慢慢跑自然而然就超過一些人到達終點了，也或許是因為路跑的路線不像操場一樣單調，所以更有動力能跑完，無論如何是一次有趣的經驗就是了。

**Comment by Elaine4488** [comment]
只有大一有參加過，記得好像是體育課規定要跑，還是有加分。那時候是跟朋友一起跑，原本設定的目標就是有完賽有運動到就好，結果我們體力差到差一點中途放棄，還好最好有堅持走完。印象比較深刻的是那一次路跑算是我第一次認真逛完校園一圈，去了很多沒去過的地方。

**Comment by Tchang323** [comment]
去年有人裝扮成恐龍路跑，今年是妝扮成郵筒，都有上校版XD，不知道明年會甚麼新玩意
